pulsar-broker/src/main/java/org/apache/pulsar/broker/service/SystemTopicTxnBufferSnapshotService.java

```diff
@@ -40,7 +43,8 @@ public class SystemTopicTxnBufferSnapshotService<T> {
 
     protected final Class<T> schemaType;
     protected final EventType systemTopicType;
-
+    protected final Map<NamespaceName, CompletableFuture<TableView<T>>> tableViewMap =
```

Comment:
We need a mechanism to evict the table view from the map if the broker doesn't own any topics of the namespace.

Reply:
Thanks for your reminder. Utilizing a timeout to expel the table view could serve as a viable stratagem.